common/lib/util/defaults.js

```diff
@@ -33,6 +33,10 @@ Defaults.getPort = function(options, tls) {
 	return (tls || options.tls) ? options.tlsPort : options.port;
 };
 
+Defaults.getHttpScheme = function(options, tls) {
```

Comment:
planned to use it for stuff like the iframetransport which needs to force tls, but didn't bother in the end (simpler to leave it as hardcoded "https://") and forgot to remove the param. oops